### ISSUE TITLE
empty if statement fails

### ISSUE THREAD

**Issue by timaschew** [js2coffee/js2coffee#265]
```
if (val) {
  // should not fail here
} else {
  foo();
}
```
fails with error **Object [object Object] has no method 'elsePartisA'**
it works on this fork: http://js2coffee.thomaskalka.de/

need to check if this was working and breaks due to missing tests

**Comment by CharlieHess** [comment]
Seeing the same behavior here. Pretty egregious, had to hunt for a while to figure out what it was failing on. :confounded:

**Comment by rstacruz** [comment]
just a note: coffeescript actually doesn't support empty ifs...

```sh
$ echo "if x == y" | coffee -s -c -b
[stdin]:1:1: error: unexpected if
```

it's happy with an empty if with an else though:

```coffee
/* echo "if x == y\nelse" | coffee -s -c -b */
if  (x === y) {

} else {

}
```

**Comment by rstacruz** [comment]
[Fixed in v2.0.0](http://js2coffee.github.io/js2coffee-redux/#try:if%20%28x%29%20{})
